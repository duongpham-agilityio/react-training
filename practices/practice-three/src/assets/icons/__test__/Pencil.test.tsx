import { render } from '@testing-library/react';

// Icons
import { Pencil } from '..';

describe('CartOutline', () => {
  it('Match to snapshot', () => {
    const { container } = render(<Pencil />);

    expect(container).toMatchSnapshot();
  });
});
