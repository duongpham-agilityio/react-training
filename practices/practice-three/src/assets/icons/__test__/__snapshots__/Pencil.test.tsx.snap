// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartOutline Match to snapshot 1`] = `
<div>
  <svg
    fill="none"
    height="24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="m18 0-3 3 6 6 3-3-6-6Zm-6 6L0 18v6h6l12-12-6-6Z"
      fill="#fff"
    />
  </svg>
</div>
`;
